### DISCUSSION TITLE
The problem is in the creation of a farming pool, I was deducted 10 xch from my balance

### DISCUSSION THREAD

**Discussion by nailhb** [Chia-Network/chia-blockchain#12572] [Support]
![Снимок](https://user-images.githubusercontent.com/109899000/180644939-db479448-546e-41d0-b2d1-6a8b29c485e0.PNG)
![Снимок
![xch](https://user-images.githubusercontent.com/109899000/180644958-42f0062d-bc08-4350-99cc-c337678582f1.PNG)
](https://user-images.githubusercontent.com/109899000/180644949-43120dc5-198a-4b4d-9216-4814ef5dd9ea.PNG)
Hello! The problem is in the creation of a farming pool, I was deducted 10 xch from my balance. All photo evidence is there. Please return 10 xch to my balance.

**Comment by nailhb** [comment]
![xch](https://user-images.githubusercontent.com/109899000/180645023-047b12c9-a5d8-4e4a-b6ab-4fca051ad677.PNG)

**Comment by nailhb** [comment]
Enter into my position do not spoil the reputation, Thank you for understanding!

**Comment by loppefaaret** [comment]
i am very sorry that this happened to you, but the transaction went through, and the fee has been paid out to the block winner.
there is no way for you to get those 10 XCH back
